packages/amplication-client/src/Resource/ResourceFormPage.tsx

```diff
@@ -35,20 +53,44 @@ function ResourceFormPage({ match }: Props) {
           <HorizontalRule />
           <TabContentTitle title="Delete Resource" />
           <Panel panelStyle={EnumPanelStyle.Error}>
-            <FlexItem itemsAlign={EnumItemsAlign.Center}>
-              <FlexItem.FlexStart>
-                <Text
-                  textColor={EnumTextColor.White}
-                  textStyle={EnumTextStyle.Description}
-                >
-                  Once you delete a resource, there is no going back. Please be
-                  certain.
-                </Text>
-              </FlexItem.FlexStart>
-              <FlexItem.FlexEnd>
-                <DeleteResourceButton resource={currentResource} />
-              </FlexItem.FlexEnd>
-            </FlexItem>
+            {!loading && (
+              <FlexItem itemsAlign={EnumItemsAlign.Center}>
+                <FlexItem.FlexStart>
+                  {resourcesCount == 0 ? (
+                    <Text
+                      textColor={EnumTextColor.White}
+                      textStyle={EnumTextStyle.Description}
+                    >
+                      Once you delete a resource, there is no going back. Please
+                      be certain.
+                    </Text>
+                  ) : (
+                    <>
+                      <Text
+                        textColor={EnumTextColor.White}
+                        textStyle={EnumTextStyle.Description}
+                      >
+                        Cannot delete a template currently used by services
+                        within the workspace. Please delete the services that
+                        are using this template, then try deleting the template
+                        again.
```

Comment:
Since we dont know yet if the user wants to delete the template (they didnt click on anything yet), change the message to :

This template is used by other resources and cannot be deleted.